presto-expressions/src/main/java/com/facebook/presto/expressions/LogicalRowExpressions.java

```diff
@@ -132,12 +132,12 @@ public static RowExpression or(RowExpression... expressions)
         return or(asList(expressions));
     }
 
-    public static RowExpression or(Collection<RowExpression> expressions)
+    public static RowExpression or(Collection<? extends RowExpression> expressions)
     {
         return binaryExpression(OR, expressions);
     }
 
-    public static RowExpression binaryExpression(Form form, Collection<RowExpression> expressions)
+    public static RowExpression binaryExpression(Form form, Collection<? extends RowExpression> expressions)
```

Comment:
It would be good to separate out to another commit before this. Many other helpers (e.g., `and`) can be changed as well.

Reply:
Done.